content/vault/v1.20.x/content/docs/updates/important-changes.mdx

```diff
@@ -192,19 +192,27 @@ audience.
 
 #### Recommendation
 
-We recommend updating your policies before Vault makes the audience value
-required for all Kubernetes authentication roles so Vault can explicitly
-validate that the audience claim in JWT tokens (`aud`) is intended for Vault and
-not another service. For example:
+We recommend configuring an audience value for Kubernetes authentication roles when possible.
+This enables Vault to validate that the aud (audience) claim in JWT tokens is explicitly intended for Vault,
+reducing the risk of token misuse by other services. While not required, setting an audience is considered a security
+best practice and is encouraged when it doesn't interfere with your workflow. For example, tokens created using
+kubernetes.io/service-account-token do not include an aud claim, so it's appropriate to omit the audience field in this case.
+Here's an example configuration.
```

Comment:
```suggestion
There are cases where configuring audience details can interfere with your
workflow. For example, tokens created using `kubernetes.io/service-account-token`
do not include an aud claim. But we recommend configuring an audience value for
Kubernetes authentication roles whenever possible. Setting explicit audience
details is best practice because it reduces the risk of token misuse by other
services. Vault can use the configured values to validate that the `aud`
(audience) claim in JWT tokens is intended for Vault.

For example:
```
Style correction: avoid "this" as a pronoun, avoid possessives